src/docs/cookbook/persistence/sqlite.md

```diff
@@ -476,9 +476,9 @@ class Dog {
 ```
 
 
-[`delete()`]: {{site.pub-api}}/sqflite/latest/sqlite_api/DatabaseExecutor/delete.html'
-[`insert()`]: {{site.pub-api}}/sqflite/latest/sqlite_api/DatabaseExecutor/insert.html
+[`delete()`]: {{site.pub-api}}/sqflite_common/latest/sqlite_api/DatabaseExecutor/delete.html'
```

Comment:
Overall, this PR looks good, but that single quite at the end of this line should be deleted.

Reply:
Ohh yeah, I missed that, Thanks, I will push the fix now

Reply:
I pushed the correction fix already, Thanks